pages/common/pio-remote.md

```diff
@@ -0,0 +1,33 @@
+# pio remote
+
+> Helper command for PlatformIO Remote Development.
+> `pio remote command` usually takes the same arguments as its counterpart `pio command`.
```

Comment:
`{{...}}` isn't rendered correctly whin used with `>`.

Reply:
I doubt that matters much though, because (1) many clients leave all of the `{{` and `}}` in pages when they display, and (2) `pio remote command` confused me.  I didn't realize that it was just a placeholder :)

Reply:
@sbrl / @navarroaxel - any opinions?

Reply:
@Waples / @nicokosi - any opinions?

Reply:
What about? :innocent: 
> `pio remote` command usually takes the same arguments as its counterpart `pio` command.

Reply:
This, or `pio remote [command]` and `pio [command]`, like it's done in man pages.

Reply:
I'm going with @marchersimon 's version.

Reply:
Because I didn't check all of them 😅

Reply:
Ah, no worries! If practically all of them do, then in this specific case, I think it's fine to drop the `usually`